examples/tutorials/astro.md

```diff
@@ -154,81 +143,127 @@ import data from "../../data/dinosaurs.json";
----
+## Add some styles
+
+You can style your app to make it your own in the `src/styles/index.css`. This
+file is imported in both the `index.astro` and `[slug].astro` files, so any
+styles you add here will apply to both pages.
+
+## Build and deploy
 
-<h1>{ name }</h1>
+Astro has a built-in command to build your site for production:
 
-<p>
-    { description }
-</p>
+```sh
+deno run build
 ```
 
-> ⚠️️ The
-> [Deno language server](https://docs.deno.com/runtime/reference/lsp_integration/)
-> does not currently support `.astro` files, so you may experience false red
-> squigglies. We're working on improving this experience.
-
-Let’s run it with `deno task dev`, and point our browser to
-`localhost:4321/abrictosaurus`:
-
-![Rendering a dynamic page for abrictosaurus](./images/how-to/astro/dynamic-page.webp)
-
-It works!
-
-## What’s next
-
-We hope this tutorial gives you a good idea of how to get started building with
-Astro and Deno. You can learn more about Astro and
-[their progressive approach to building websites](https://docs.astro.build/en/getting-started/).
-If you’re interested in swapping out our hardcoded data store, here are some
-resources on
-[connecting to databases with Deno](https://docs.deno.com/runtime/tutorials/connecting_to_databases/),
-including
-[Planetscale](https://docs.deno.com/runtime/tutorials/how_to_with_npm/planetscale/),
-[Redis](https://docs.deno.com/runtime/tutorials/how_to_with_npm/redis/), and
-more. Or you can learn how to
-[deploy your Astro project to Deno Deploy](https://deno.com/blog/astro-on-deno),
-or follow these guides on how to self-host Deno to
-[AWS](https://docs.deno.com/runtime/tutorials/aws_lightsail/),
-[Digital Ocean](https://docs.deno.com/runtime/tutorials/digital_ocean/), and
-[Google Cloud Run](https://docs.deno.com/runtime/tutorials/google_cloud_run/).
+This will:
+
+- Generate static HTML files for each page in the `dist` directory.
+- Optimize your assets (CSS, JavaScript, images, etc.) for production.
+
+You can deploy this app to your favorite cloud provider. We recommend using
+[Deno Deploy](https://deno.com/deploy) for a simple and easy deployment
+experience. You can deploy your app directly from GitHub, simply create a GitHub
+repository and push your code there, then connect it to Deno Deploy.
+
+### Create a GitHub repository
+
+[Create a new GitHub repository](https://github.com/new), then initialize and
+push your app to GitHub:
+
+```sh
+git init -b main
+git remote add origin https://github.com/<your_github_username>/<your_repo_name>.git
+git add .
+git commit -am 'initial commit'
+git push -u origin main
+```
+
+### Deploy to Deno Deploy
+
+Once your app is on GitHub, you can deploy it on the Deno Deploy<sup>EA</sup>
+dashboard.
+<a href="https://app.deno.com/" class="docs-cta deploy-cta deploy-button">Deploy
+my app</a>
+
+For a walkthrough of deploying your app, check out the
+[Deno Deploy tutorial](/examples/deno_deploy_tutorial/).
```

Comment:
I think we should have one clear Deploy button here with some standard language to tell the reader that they can quickly clone and deploy what they just read by clicking the deploy button.

And we should replace this rounded CTA button with a link to the deploy console so there is no confusion int he CTAs

Reply:
It may also be worth adding some standard preamble at the top of the tutorial with a Deploy button to give people the option of following the steps, or using the deploy button to clone this so that they can explore the finished article.